Resources/config/doctrine/model/AuthorizationCode.orm.xml

```diff
@@ -14,6 +14,11 @@
         <field name="userIdentifier" type="string" length="128" nullable="true" />
         <field name="scopes" type="oauth2_scope" nullable="true" />
         <field name="revoked" type="boolean" />
+        <indexes>
+            <index name="idx_oauth2_authorization_code_client" columns="client"/>
```

Comment:
@franjo-zadelj-trikoder this is redundant, doctrine relationships already get an index by default